From 8464ea4aa20182d3f56c06287cbb8a2ff71e2837 Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Thu, 14 Jul 2016 15:36:03 -0700
Subject: [PATCH 14/15] Ticket 48925 - slapd crash with SIGILL: Dsktune should
 detect lack of CMPXCHG16B

Description: escaping '"' in the ERROR message.

Reviewed by: one-line rule

(cherry picked from commit 975e0fae044a08d2755b2394bac09d722bd1bae0)
---
 ldap/systools/idsktune.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/systools/idsktune.c b/ldap/systools/idsktune.c
index b6c352a..ad6a39e 100644
--- a/ldap/systools/idsktune.c
+++ b/ldap/systools/idsktune.c
@@ -895,7 +895,7 @@ linux_check_cpu_features(void)
     if (found == 0) {
         flag_os_bad = 1;
         printf("ERROR: This system does not support CMPXCHG16B instruction (cpuflag cx16).\n");
-        printf("       nsslapd-enable-nunc-stans must be set to "off" on this system. \n");
+        printf("       nsslapd-enable-nunc-stans must be set to \"off\" on this system. \n");
         printf("       In a future release of Directory Server this platform will NOT be supported.\n\n");
     }
 
-- 
2.4.11

